.eslintrc.js

```diff
@@ -89,6 +89,10 @@ module.exports = {
             message:
               "Use 'useLocation', 'useParams', 'useNavigate', 'useRoutes' from sentry/utils instead.",
           },
+          {
+            name: 'qs',
+            message: 'Please use query-string instead of qs',
+          },
```

Comment:
should this be in biome lol

Reply:
we could port them all to biome, didn't know we were using them there